src/lib.rs

```diff
@@ -224,6 +214,8 @@ pub use {
     blocking_reader::{BlockingRawBinaryReader, BlockingRawReader, BlockingRawTextReader},
     ion_reader::IonReader,
     raw_reader::{BufferedRawReader, RawReader, RawStreamItem},
+    raw_symbol_token::RawSymbolToken,
```

Comment:
I'm curious. Do you think it would be helpful, neutral, or unhelpful for everything in the `experimental-reader` feature to be exported under `ion_rs::reader` as a way of making the namespaces somewhat match the features?

Reply:
It depends -- if we're leaning towards a pretty flat namespace, I don't think it would help that much. If we think we'll ultimately have visible modules, it might be somewhat helpful.

Reply:
I don't know if we're leaning towards a flat namespace. I do think that the most common things should probably be at the front and center.

That being said, I think figuring this out would be a blocking issue for stabilizing the reader feature, but not for the v1.0 release.